### PR TITLE
generate verbatim sheets and attempt round trip

### PR DESCRIPTION
**Pull request by turbomam**
for #41 

As previously demonstrated. I improved the click CLIs and am just requesting a merge now.

### CODE REVIEW

File: schemasheets/get_metaclass_slotvals.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+@click.option("--selected_element", required=True)
+@click.option(
+    "--directory",
+    required=True,
+    help="Destination directory element reports",
+    default="element_reports",
+)
+def cli(selected_element: str, directory: str):
+    """
+    :param selected_element:
```
Comment (cmungall) [inline schemasheets/get_metaclass_slotvals.py:33]: don't use ReST docs for CLI commands

Add a description of what this does

File: schemasheets/verbatim_sheets.py [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+logger = logging.getLogger(__name__)
+click_log.basic_config(logger)
+
+pd.set_option("display.max_columns", None)
+
+# todo deep roundtrip diff
+#  remember there are cowardly skips
+#  And know elements that don't propagate over imports like subsets and prefixes
+
+cowardly_skip = [
```
Comment (cmungall) [inline schemasheets/verbatim_sheets.py:25]: cute name but consider something that helps explain what it is :-)

File: schemasheets/get_metaclass_slotvals.py [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    #   - linkml:extensions
+    #   - linkml:annotations
+
+    # annotations.yaml
+    # extensions.yaml
+    # mappings.yaml
+    # types.yaml
+
+
+def element_to_is_dict(view: SchemaView, element):
```
Comment (cmungall) [inline schemasheets/get_metaclass_slotvals.py:160]: consider putting these before the cli section

**Comment by turbomam** [comment]
> @turbomam what's the status of this?

I guess we can close it. preparing one or more templates for linkml2sheets isn't too onerous.

**Pull request closed without merging**